receiver/elasticsearchreceiver/documentation.md

```diff
@@ -49,20 +49,20 @@ metrics:
 
 ## Metric attributes
 
-| Name | Description |
-| ---- | ----------- |
-| cache_name | The name of cache. |
-| collector_name | The name of the garbage collector. |
-| direction | The direction of network data. |
-| disk_usage_state | The state of a section of space on disk. |
-| document_state | The state of the document. |
-| elasticsearch.cluster.name | The name of the elasticsearch cluster. |
-| elasticsearch.node.name | The name of the elasticsearch node. |
-| fs_direction | The direction of filesystem IO. |
-| health_status | The health status of the cluster. |
-| memory_pool_name | The name of the JVM memory pool. |
-| operation | The type of operation. |
-| shard_state | The state of the shard. |
-| task_state | The state of the task. |
-| thread_pool_name | The name of the thread pool. |
-| thread_state | The state of the thread. |
+| Name | Description | Values |
+| ---- | ----------- | ------ |
+| cache_name | The name of cache. | fielddata, query |
+| collector_name (name) | The name of the garbage collector. |  |
```

Comment:
Is there a value in the data in parentheses? It seems the name always includes the same information. I see further down some other metrics (mysql receiver) don't have the suffix, should they include it? 

The question is really around consistency of the metrics being generated. I'm happy to have this answered in a separate issue though.